tests/test_record.py

```diff
@@ -0,0 +1,45 @@
+__author__ = "Bailey Nozomu Hwa <hwab@janelia.hhmi.org>"
+__date__ = "$Aug 10, 2015, 3:47 PM$"
+
+import os
+import shutil
+import tempfile
+import RPi.GPIO as GPIO
+import threading
+
+from mouse.record import main
+
+
+class TestPicture(object):
+    def setup(self):
+        self.cwd = os.getcwd()
+        self.tempdir = ""
+        self.tempdir = tempfile.mkdtemp()
+        os.chdir(self.tempdir)
+
+        print("tempdir = \"%s\"" % self.tempdir)
+
+    def teardown(self):
+        os.chdir(self.cwd)
+        shutil.rmtree(self.tempdir)
+        self.tempdir = ""
+        self.cwd = ""
+
+    def test_main_0(self):
+        t = threading.Thread(target=main, args=(
+            "",
+            "1",
+            "1",
+            "."
+        ))
+        t.daemon = True
+        t.start()
+        t.join(2)
+        GPIO.output(27, (GPIO.LOW, GPIO.HIGH, GPIO.LOW))
```

Comment:
From our conversation earlier...

I think this needs to be broken out into three statements. Each one taking a single `GPIO.LOW` or `GPIO.HIGH` value.